packages/amplication-data-service-generator/src/update-package-jsons.ts

```diff
@@ -12,64 +12,27 @@ import { merge } from "lodash";
 export function updatePackageJSONs(
   modules: Module[],
   baseDirectory: string,
-  update: Record<string, any>
+  update: { [key: string]: any }[]
 ): Module[] {
   return modules.map((module) => {
-    if (module.path === `${baseDirectory}/package.json`) {
-      return updatePackageJSON(module, update);
-    } else if (module.path === `${baseDirectory}/package-lock.json`) {
-      return updatePackageLockJSON(module, update);
-    }
-    return module;
-  });
-}
-
-function updatePackageJSON(module: Module, update: Record<string, any>) {
-  let pkg = JSON.parse(module.code);
-  pkg = merge(pkg, update);
-
-  if (!semver.valid(pkg.version)) {
-    delete pkg.version;
-  }
+    const updateModule =
+      module.path === `${baseDirectory}/package.json`
+        ? updatePackageJSON(module, update)
+        : module;
```

Comment:
 since we remove the logic for package-lock I don't think we need this condition at all. 

```suggestion
        updatePackageJSON(module, update);
```

Reply:
Actually, it seems like we do have to check the path.
After I made this change, I ran the tests and they failed. 
I looked at it with the debugger and it seems like it run this function on other paths that we don't want

![image](https://user-images.githubusercontent.com/39680385/195537019-6d5b8ae0-b1c8-4577-8b21-106739e20ae9.png)

Reply:
with the check, it calls the function only when the path is `${baseDirectory}/package.json` and then the tests pass
![image](https://user-images.githubusercontent.com/39680385/195537601-7206037b-0fe5-4303-82f0-9cf3885619bd.png)

Reply:
@abrl91 there is no reason to send this function any file other than the files that needed to be manipulated. 

I looked at the code and noticed that in the admin-ui we send the function ALL the static files, instead of filtering the files.
This behavior need to be fixed - and then we will be able to remove the condition  

<img width="1006" alt="image" src="https://user-images.githubusercontent.com/43705455/195973716-b4a74c9b-1048-40f6-9f20-f33ef8ca7233.png">

Reply:
Got it. I will fix it